packages/react-vapor/jest/identity-obj-proxy-esm.js

```diff
@@ -0,0 +1,11 @@
+module.exports = new Proxy(
```

Comment:
I think the jest folder was moved from `.jest` to `jest`. But I don't see the equivalent deleted file in this PR

Reply:
I remember having to resove something about the jest path with @aiheon about that.  The tests are running fine in the branch so I guess we got it right.

Reply:
![screenshot_113](https://user-images.githubusercontent.com/45688129/108255633-893d6780-712a-11eb-928a-b14c26c85032.png)
good catch Germain